### PR TITLE
Moved requirements

### PR DESCRIPTION
**Pull request by hietkamp**
Requirements of rfc009 are moved to the bolts. RFC describes now the implementation of DID's and VC's as specified by W3C.

### CODE REVIEW

File: rfc/rfc009-verifiable-presentations.md [hunk 1]
```diff
@@ -92,308 +34,21 @@ The terminology used is defined in https://www.w3.org/TR/did-core/#terminology a
...
-  "credentialStatus": {
-    "id": "https://example.com/status/<status list identifier>",
-    "type": "CredentialStatusList2017"
-  }
-```
-
-The credentials list is specified by [Credential Status List (2017)](https://w3c-ccg.github.io/vc-csl2017/).
+## 3. Requirements for verifiable credentials
 
+The Nuts framework MUST support the use of verifiable credentials in the roles of Issuer, Holder, and Verifier as described in [Verifiable Credentials Use Cases](https://www.w3.org/TR/vc-use-cases/) and [W3C Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/). In the terms of decentralized Identifiers, the Nuts framework acts as a software agent of the person or organisation accountable for the role of Issuer, Holder, and Verifier.
```
Comment (woutslakhorst) [inline rfc/rfc009-verifiable-presentations.md:39]: I think you refer to `An implementation` in the context of a piece of software that implements these RFCs? The term framework is not yet used and I don't see how it's applicable.

Comment (woutslakhorst) [inline rfc/rfc009-verifiable-presentations.md:39]: Holder and Verifier sounds logical. I think the Issuer part is optional. Not every node/implementation MUST be able to issue VCs. Unless I'm missing a use-case?

File: rfc/rfc009-verifiable-presentations.md [hunk 2]
```diff
@@ -92,308 +34,21 @@ The terminology used is defined in https://www.w3.org/TR/did-core/#terminology a
...
 
+The Nuts framework MUST support the use decentralized identifiers as described in https://www.w3.org/TR/did-core/.
```
Comment (woutslakhorst) [inline rfc/rfc009-verifiable-presentations.md:41]: This sounds very abstract? How can something be supported that defines a framework for specs that can be supported.
eg: you could say that we MUST support the resolving of web dids.

File: rfc/rfc009-verifiable-presentations.md [hunk 3]
```diff
@@ -92,308 +34,21 @@ The terminology used is defined in https://www.w3.org/TR/did-core/#terminology a
...
+The Nuts framework MAY have limited support on the following issues:
 
-Downloaded list of trusted issuers.
+- The framework supports the JSON Web Keys (JWK) as key format (verification method type equals `JsonWebKey2020`). See also https://www.w3.org/TR/did-spec-registries/.
+- The framework has limited support for key types (minimal OKP) and algorithms (minimal EdDSA).
+- The framework has limited support for DID methods (only web method), which resolve to a DID document in a verifiable data registry. See https://w3c-ccg.github.io/did-method-web/.
 
+The Nuts framework MUST implement a secure Wallet to store decentralized Identifiers (DID) and their private key, and to store verifiable credentials that the agent holds.
 
+The Nuts framework MAY introduce 'irma' as did method to support the IRMA signing and verification.
```
Comment (woutslakhorst) [inline rfc/rfc009-verifiable-presentations.md:51]: I guess this can be removed and be added as separate RFC or part of a RFC when needed?

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: rfc/rfc009-verifiable-presentations.md (hunk 1 above)
Comment (hietkamp) [reply rfc/rfc009-verifiable-presentations.md:39]: Agreed

Comment (hietkamp) [reply rfc/rfc009-verifiable-presentations.md:39]: The issuer part is optional now. I think its a priority issue.

File: rfc/rfc009-verifiable-presentations.md (hunk 2 above)
Comment (hietkamp) [reply rfc/rfc009-verifiable-presentations.md:41]: The implementation must support the core to be able to support the web method. Will change it to that.

File: rfc/rfc009-verifiable-presentations.md (hunk 3 above)
Comment (hietkamp) [reply rfc/rfc009-verifiable-presentations.md:51]: Agreed.

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**